pkgs/servers/nas/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, imake, bison, flex, gccmakedep, xorg }:
+
+let
+  pname = "nas";
+  version = "1.9.4";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.src.tar.gz";
+    sha256 = "17dk0ckm6mp1ajc0cd6bwyi638ynw2f6bhbn7gynrs0wfmiyldng";
+  };
+
+  buildInputs = [ imake bison flex gccmakedep ]
+  ++ (with xorg; [ xproto libXau libXt libXext libXaw libXpm ]);
+
+  buildPhase = ''
+    xmkmf
```

Comment:
I've found that `make WORLDOPTS="" World` is needed to ensure there are no build errors (default is `WORLDOPTS=-k` in Imakefile) and this only works for me using `buildPhase`